### ISSUE TITLE
Flogo-web builds dynamically linked executables

### ISSUE THREAD

**Issue by retgits** [TIBCOSoftware/flogo#313]
**I'm submitting a ...**  (check one with "x")
```
[X] bug report => search github for a similar issue or PR before submitting
[] feature request
[] support request
[] general question
```

**Current behavior (how does the issue manifest):** When you're designing an app in the Flogo Web UI and export it using the "Linux/amd64" option, it generates a dynamically linked binary file.

**Expected behavior:** To make the app run on any type of Linux system, it should be a statically linked binary (this isn't an issue with either macOS or Windows, it will only show up on Linux)

**Minimal steps to reproduce the problem (not required if feature enhancement):**

* Create an app
* Build as a "Linux/amd64" app
* Run the app on a Linux machine (e.g. Ubuntu)

The output of the file command:
```
$ file my-app_linux_amd64 
my-app_linux_amd64: ELF 64-bit LSB executable, x86-64, version 1 (SYSV), dynamically linked, interpreter /lib/ld-musl-x86_64.so.1, with debug_info, not stripped
```

`musl` is not installed by default on all Linux systems and we shouldn't require users to do so.

**What is the motivation / use case for changing the behavior?** It breaks compatibility with Linux systems

**Please tell us about your environment (Operating system, docker version, browser & web ui version, etc):** Web UI Flogo v0.5.4

**Additional information you deem important (e.g. issue happens only occasionally):**

The issue can be fixed relatively easily by adding `CGO_ENABLED=0` to the command line when building the app (this will disable CGO builds). Having CGO enabled will generate dynamically linked binaries for Linux systems

**Comment by haarchri** [comment]
the problem is fixed in the last build 0.5.6 - we tested with rhel7 is working now

**Comment by retgits** [comment]
Closing as the issue was fixed
